package.json

```diff
@@ -92,6 +92,11 @@
         "title": "New Component",
         "category": "OpenShift"
       },
+      {
+        "command": "openshift.component.delete.palette",
```

Comment:
AFAIK, this does not need to be a separate command - why not use the existing component.delete?

Reply:
The name in context menu 'Delete' if we use the same command for palette and context menu we end up having bunch of `OpenShift: Delete' commands. One per model object.

Reply:
So, if the existing command was renamed to, say, 'Delete Component', would that be a problem? I'm guessing it would show as such in the context menu as well.